### ISSUE TITLE
vulnerability alerts

### ISSUE THREAD

**Issue by hdrodel** [temsa/qs-google-signature#2]
i'm using the google-distance-matrix npm package. i'm getting vulnerability alerts because of the qs version and mocha version. 
I [tried with newer versions](https://github.com/hdrodel/qs-google-signature/blob/master/package.json) what seems to fix the issue i think? 

anyone els getting the same alerts?

**Comment by NishchayGupta** [comment]
Yes, I am getting the same error. Is the @author (@temsa) working on the fix?
